Lib/test/test_builtin.py

```diff
@@ -432,30 +428,41 @@ async def arange(n):
             for i in range(n):
                 yield i
 
+        class Lock:
+            async def __aenter__(self):
+                return self
+
+            async def __aexit__(self, *exc_info):
+                pass
+
+        async def sleep(delay, result=None):
+            await async_yield(delay)
```

Comment:
Not for this PR but impl of `async_yield` should check that delay must be 0 here.

Reply:
I plan to use async yield in other tests, I can make the check here